docker/docs/configuration.dev.md

```diff
@@ -4,25 +4,19 @@
 ## Index
```

Comment:
I think it would be better to state that this file is a work in progress. Because there are multiple broken links.

Reply:
deleted